src/api/VoteMonitor.Api.Location/Services/XslxFileLoader.cs

```diff
@@ -12,10 +13,26 @@ namespace VoteMonitor.Api.Location.Services
     public class XlsxFileLoader : IFileLoader
     {
         private static readonly string[] SUFFIXES = { ".xlsx", ".xls" };
-        private static readonly int COD_JUDET_INDEX = 1;
-        private static readonly int ADRESA_INDEX = 7;
-        private static readonly int SIRUTA_INDEX = 3;
-        private static readonly int NR_SECTIE_INDEX = 5;
+        private static PollingStationExcelHeader header = new PollingStationExcelHeader();
+
+        public byte[] ExportHeaderInformation()
+        {
+            using (var excelPackage = new ExcelPackage())
```

Comment:
Can we use "[using declaration](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-8#using-declarations)" instead of this using statement to improve the readability?

Reply:
Sure, thanks for pointing this out! Meanwhile I brought the change, should be fine now - please confirm that we have the same understanding.